src/main/java/com/rakovets/course/java/core/practice/jcf_set/web_service/baseOfClients.java

```diff
@@ -0,0 +1,53 @@
+package com.rakovets.course.java.core.practice.jcf_set.web_service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class baseOfClients {
+    private Map<String,String> data = new HashMap<>();
+
+    @Override
+    public String toString() {
+        return "data{" + "data =" + data + '}';
+    }
+
+    public baseOfClients(Map<String,String> data){
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"